src/App.tsx

```diff
@@ -1,57 +1,23 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
+import { Dropdown } from './components/Dropdown';
 import { peopleFromServer } from './data/people';
+import { Person } from './types/Person';
 
 export const App: React.FC = () => {
-  const { name, born, died } = peopleFromServer[0];
+  const [onSelected, setOnSelected] = useState<Person | null>(null);
```

Comment:
```suggestion
  const [selectedPerson, setSelectedPerson] = useState<Person | null>(null);
```
Try to provide more meaningful names